lib/galaxy/config/__init__.py

```diff
@@ -363,6 +363,10 @@ def _process_config(self, kwargs):
                 log.error("CONFIGURATION ERROR: Can't open supplied blacklist file from path: " + str(self.blacklist_file))
 
         self.persistent_communication_rooms = listify(kwargs.get("persistent_communication_rooms", []), do_strip=True)
+        self.enable_tool_recommendations = string_as_bool(kwargs.get('enable_tool_recommendations', 'False'))
```

Comment:
Once these config options are added to `config_schema.yml` (see other comment), corresponding properties will be generated automatically (and converted to correct datatype). See this: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L191 - these 5 lines summarize what the new config does (it's a very recent modification).